webapp/channels/src/components/admin_console/system_users/system_users_date_range_selector/index.tsx

```diff
@@ -0,0 +1,202 @@
+                                    endDate: formatDate(now),
+                                }}
+                            />
+                        </>
+                    }
+                    trailingElements={inputValue === ReportDuration.Last30Days && <i className='icon icon-check'/>}
+                    onClick={() => updateDateRange(ReportDuration.Last30Days)}
+                />
+                <Menu.Item
+                    key={ReportDuration.PreviousMonth}
+                    id={ReportDuration.PreviousMonth}
+                    role='menuitem'
+                    labels={
+                        <>
+                            <FormattedMessage
+                                id='admin.system_users.date_range_selector.date_range.previous_month'
+                                defaultMessage='Previous month'
+                            />
+                            <FormattedMessage
+                                id='admin.system_users.date_range_selector.date_range.sublabel'
+                                defaultMessage='{startDate} - {endDate}'
+                                values={{
+                                    startDate: formatDate(getBeginningOfLastMonth(now)),
+                                    endDate: formatDate(getEndOfLastMonth(now)),
+                                }}
+                            />
+                        </>
+                    }
+                    trailingElements={inputValue === ReportDuration.PreviousMonth && <i className='icon icon-check'/>}
+                    onClick={() => updateDateRange(ReportDuration.PreviousMonth)}
+                />
+                <Menu.Item
+                    key={ReportDuration.Last6Months}
+                    id={ReportDuration.Last6Months}
+                    role='menuitem'
+                    labels={
+                        <>
+                            <FormattedMessage
+                                id='admin.system_users.date_range_selector.date_range.last_6_months'
+                                defaultMessage='Last 6 months'
+                            />
+                            <FormattedMessage
+                                id='admin.system_users.date_range_selector.date_range.sublabel'
+                                defaultMessage='{startDate} - {endDate}'
+                                values={{
+                                    startDate: formatDate(get6MonthsBack(now), {month: 'numeric', year: 'numeric'}),
+                                    endDate: formatDate(now, {month: 'numeric', year: 'numeric'}),
+                                }}
+                            />
+                        </>
+
+                    }
+                    trailingElements={inputValue === ReportDuration.Last6Months && <i className='icon icon-check'/>}
+                    onClick={() => updateDateRange(ReportDuration.Last6Months)}
+                />
+                <hr/>
```

Comment:
you can use Menu.Divider

Reply:
I'm not seeing a reference to `Menu.Divider`, is is somewhere else not in `components/menu`?

Reply:
sorry Seperator 😛

Reply:
done